src/dotnet/commands/dotnet-test/xlf/LocalizableStrings.pt-BR.xlf

```diff
@@ -74,9 +74,9 @@
       <trans-unit id="CmdLoggerDescription">
         <source>Specify a logger for test results. 
                                         Example: --logger "trx[;LogFileName=&lt;Defaults to unique file name&gt;]"</source>
-        <target state="needs-review-translation">Especifique um agente para os resultados de teste. 
-                                        Exemplo: --logger:trx</target>
-        <note></note>
+        <target state="translated">Especifique um agente para os resultados de teste. 
+                                        Exemplo: --logger "trx[;LogFileName=&lt;Defaults to unique file name&gt;]"</target>
```

Comment:
This is missing translation to the string in the middle: Defaults to unique file name